osu.Game/Input/Bindings/GlobalActionContainer.cs

```diff
@@ -442,6 +466,7 @@ public enum GlobalActionCategory
         Replay,
         SongSelect,
         AudioControl,
-        Overlays
+        Overlays,
+        EditorTestPlay,
```

Comment:
These getting a separate section will likely look gratuitous but it bypasses several issues:

- If they were to be implemented as non-rebindable, they wouldn't be listed anywhere, so there'd probably need to be text that explains that these exist
- <kbd>F1</kbd> and <kbd>F2</kbd> are already used in the editor section (for compose/design tabs), and keybindings must be unique within a section
- This also solves various input priority weirdness. As it stands, `KeyBindingContainer` takes precedence over plain `OnKeyDown()` which meant that other various bindings like leaderboard toggle were eating these hotkeys when implementing them via `OnKeyDown()`.